code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -174,8 +175,8 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/roller = 1,
 		/obj/item/spear = 1,
 		/obj/item/weldingtool/largetank = 1,
-		/obj/item/gun/ballistic/automatic/pistol = 1,
-		/obj/item/gun/ballistic/automatic/pistol/m1911 = 1,
+		/obj/item/gun/ballistic/automatic/pistol/divet/rb = 1,
+		/obj/item/gun/ballistic/automatic/pistol/divet/no_mag = 1,
```

Comment:
Put these in the "weapons and ammo" section you made, since guns aren't tools technically.